pages/searching/time_frame_selector.rst

```diff
@@ -1,38 +1,96 @@
-Time frame selector
+Time Frame Selector
 -------------------
 
-The time frame selector defines in what time range to search in. It offers three different ways of selecting a time range and
-is vital for search speed: If you know you are only interested in messages of the last hour, only search in that time frame.
-This will make Graylog search in :doc:`relevant indices </pages/configuration/index_model>` only and greatly reduce system load and required resources.
+The Time frame selector allows you to pull specific time ranges from your Graylog data. 
+This feature will help you and other Graylog users analyze issues that affect multiple aspects 
+of your environment. Most importantly, the tool offer multiple ways to filter time ranges 
+you need for success.
 
-.. image:: /images/searching/queries_time_range_selector.png
-   :align: center
+.. image:: /images/searching/time_frame_selector_wide.png
+
+
+To that end, the tool's ability to help you queries can help you perform such actions as:
+
+* understanding and responding to data breaches, broken processes, and other security incidents
+* troubleshoot systems and networks
+* understand the behaviors of your users
+* conducting forensics activities
+  
+
+Time Frame Options
+^^^^^^^^^^^^^^^^^^
+You can access the Selector from the Search menu.
+
+To access the window, click the clock icon. A dialog pops up, offering the following ranges:
+
+* Relative
+* Absolute
+* Keyword
 
-.. _relative-time-frame-selector:
 
 Relative time frame selector
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-The relative time frame selector lets you look for messages from the selected option to the time you hit the search button. The selector
-offers a wide set of relative time frames that fit most of your search needs.
+The relative time frame selector lets you look for messages from the selected option to the time you 
+hit the search button. The selector offers a wide set of relative time frames that fit most of your 
+search needs, including an *All Time* option.
```

Comment:
```suggestion
The relative time frame selector lets you search for messages within time ranges relative to ``now`` or another date of your choosing. The selector offers a wide set of relative time frames that fit most of your search needs, including an *All Time* option.
```
I know this existed before, but the wording felt like it could be improved. This previous part was not fully correct: "to the time you hit the search button". My wording suggestion is still not that great. Perhaps you have another suggestion.

Reply:
Added a minor update to your suggestion.